src/main/java/dev/openfga/sdk/api/OpenFgaApi.java

```diff
@@ -129,11 +126,18 @@ private CompletableFuture<ApiResponse<CheckResponse>> check(
 
         String path = "/stores/{store_id}/check".replace("{store_id}", ApiClient.urlEncode(storeId.toString()));
 
+        Map<String, Object> methodParameters = new HashMap<>();
+        methodParameters.put("storeId", storeId);
+        methodParameters.put("body", body);
+
+        Map<Attribute, String> telemetryAttributes = buildTelemetryAttributes(methodParameters);
```

Comment:
Really like this way of handling it, makes it super easy to roll new attributes out and ensure alignment

Reply:
Thanks very much!